pupil-spa/Dockerfile

```diff
@@ -12,7 +12,7 @@ COPY package.json .
 COPY yarn.lock .
 
 ## Storing node modules on a separate layer will prevent unnecessary npm installs at each build
-RUN yarn install --frozen-lockfile --ignore-engines && mkdir /ng-app && cp -R ./node_modules ./ng-app
+RUN yarn install --frozen-lockfile --ignore-engines --production && mkdir /ng-app && cp -R ./node_modules ./ng-app
```

Comment:
I suspect this breaks the Docker build.  No issue if it doesn't though.

Reply:
yeah i've got a lot of broken scripts, as i overlooked the `gulp` and `tsc` dependencies.

May have to rework some of them with more intermediary containers 🤔